packages/cozy-ccc-libs/src/ContentScript.js

```diff
@@ -0,0 +1,454 @@
+import get from 'lodash/get'
```

Comment:
miss the package.json / lock with this commit. Today this commit is not functional